### DISCUSSION TITLE
Reducing file size/compression of files

### DISCUSSION THREAD

**Discussion by Ryuk-hash** [strapi/strapi#6952] [Legacy Topics]
Hi guys.
**Issue:** Right now file upload is taking a lot of time. So I have a settings.js file in the extensions > upload folder whereas in there I've defined module.exports with aws details for different environments. Where, or in which file exactly do I make changes for the following problem?

_**How can I implement reducing file size/compression in strapi upload plugin?**_

![image](https://user-images.githubusercontent.com/50495162/87003199-d2aef700-c1d8-11ea-9f9f-e98aeeb4f320.png)

Kindly let me know, thanks in advance!

**Comment by derrickmehaffy** [comment]
I'm guessing that maybe this relates to the media library doing some conversions automatically? In your admin under the settings option you can disable this (this setting is currently stored in the database)

**Reply by Ryuk-hash** [reply]
Can you elaborate please? I don't understand the answer

**Comment by Ryuk-hash** [comment]
Anyone please help?
